sdk/formrecognizer/azure-ai-formrecognizer/pom.xml

```diff
@@ -48,7 +48,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.2.1</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-test;unreleased_package} -->
```

Comment:
This is using the wrong version tag. You should add an `unreleased_com.azure:azure-core-test;1.3.0-beta.1` into `version_client.txt` and then change this tag to

```
<!-- {x-version-update;unreleased_com.azure:azure-core-test;dependency} -->
```